nncf/common/quantization/quantizer_propagation/graph.py

```diff
@@ -769,6 +769,38 @@ def recursive_helper(curr_node_key: str, target_node_list: List[str]):
         recursive_helper(node_key, ret_node_key_list)
         return ret_node_key_list
 
+    def all_outputs_are_quantized(self, node_key) -> bool:
+        """
+        Returns True if all pathes from the given node to the first
+        input quantable nodes have an activation quantizer, False otherwise.
+
+        :param node_key: Given node key.
+        :return: True if all pathes from the given node to the first
+        input quantable nodes have an activation quantizer, False otherwise.
+        """
+
+        def recursive_helper(curr_node_key: str):
```

Comment:
From the cases based on the `BiasCorrection` algorithm, the recursion is not the better option since it has limited depth and the output case may not be reached in optimal time. 
Is it possible to use the `while` option here?
Major.